core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt

```diff
@@ -123,7 +123,7 @@ abstract class PageFragment : OnItemClickListener, BaseFragment(), FragmentActiv
     activity.setSupportActionBar(toolbar)
     activity.supportActionBar?.apply {
       setDisplayHomeAsUpEnabled(true)
-      title = title
+      title = this@PageFragment.title
```

Comment:
make sure to ignore the warning that is calling this `this` unnecessary or some unsuspecting good samaritan could easily reintroduce this bug

Reply:
@macgills I don't know why is this calling `this` is unnecessary? Do you want me to suppress this warning on deket?

Reply:
option 2 if we remove the title inside of the toolbar scope. which one is better?

Reply:
the warning is very useful so we do not want to disable it, also it is not from detekt, it is an inspection. press alt+enter on it and you will get suppression options, you can suppress this statement only.

This is an unfortunate false positive, alternatively we could rename title and call it `screenTitle` or the like